<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectResourceDialog</class>
 <widget class="QDialog" name="SelectResourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>270</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Añadir Solicitud de un Proceso</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/man-in-the-deadlock-1.png</normaloff>:/images/man-in-the-deadlock-1.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seleccionar Recursos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recurso:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxResource">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinResourceNum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Añadir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>27</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Atrás</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAccept">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>27</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recursos Seleccionados:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="boxSelectedResources">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>210</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
